pkg/image/utilities/utilities.go

```diff
@@ -622,52 +650,62 @@ if ($help)
     exit 0
 }
 
-if (-not $registry)
+if (-not $target_registry)
 {
-	echo "Registry address is required"
+    echo "Registry address is required"
     usage
     exit 1
 }
 
 if (-not (Test-Path $images))
 {
-	echo "Could not find '$images'"
+    echo "Could not find '$images'"
     usage
-	exit 1
+    exit 1
 }
 
 docker load --input $images
 if (-not $?)
 {
-	echo "Could not load '$images'"
+    echo "Could not load '$images'"
     exit 1
 }
 
 if (-not (Test-Path $image_list))
 {
-	exit 0
+    exit 0
+}
+
+if ([string]::IsNull($source_registry))
+{
+    $source_registry = ""
+}
+elseif (-not [string]::IsNullOrEmpty($source_registry))
+{
+    $source_registry += "/"
 }
 
 Get-Content -Force -Path $image_list | ForEach-Object {
     if ($_) {
         $fullname_image = ('{0}-windows-{1}' -f $_, $os_release_id)
-		echo "Tagging $registry/$fullname_image"
-	
-		switch -regex ($fullname_image)
-		{
-			'.+/.+' {
-				docker tag $fullname_image $registry/$fullname_image
+        $source_image = -join ($source_registry, $fullname_image)
```

Comment:
```suggestion
        $source_image = -join ($source_registry, "/", $fullname_image)
```

not sure if this is the right syntax (can `-join` take 3 args?) but as mentioned above assume `source_registry` does not have a trailing slash

Reply:
also, should we just add this into `$fullname_image`'s definition instead and continue to use that? Why modify the `$source_image`?

Reply:
Adding a slash here breaks the scenario where source registry is empty (i.e. Docker Hub).

Reply:
The idea is to keep `$fullname_image` as it was originally so we don't need to parse registry information in the script, but rather just prepend the new registry name.

Reply:
Updated `$fullname_image` as requested.